test/release/examples/hello5-taskpar.chpl

```diff
@@ -1,36 +1,40 @@
-/*  This test uses Chapel's task parallel features to create a
- *  parallel hello world program that utilizes multiple cores on a
- *  single locale (node)
+// Task-parallel hello world
+
+/* This program uses Chapel's `task parallel` features to express an
+   explicitly concurrent hello world program that utilizes multiple
+   cores on a single `locale` (compute node).
  */
 
 
 //
-// Specify the number of tasks to create via a config const.  By
-// default, we use the tasking layer's estimate of maximum parallelism
-// on the current locale ('here').  This default can be overridden on
-// the executable command line (e.g., using --numTasks=3).
+// First, we specify the number of tasks to create via a `config
+// const`.  By default, set it to the runtime's estimation of maximum
+// parallelism that the current locale ('`here`') is capable of
+// executing (``.maxTaskPar``).
 //
 config const numTasks = here.maxTaskPar;
 
 
 //
-// Create the requested tasks using a 'coforall' loop which will
-// create a unique task per iteration.  Here, we're iterating over
-// 0..#numTasks which represents the range starting at 0 and
-// containing 'numTasks' members (equivalent to 0..numTasks-1).
-// Thus, the loop index variable 'tid' will take on unique values
-// in the range 0..numTasks-1.
+// Next, we create the specified number tasks using a `coforall-loop`.
+// This is a parallel loop form that will create a distinct task per
+// iteration.
+//
+// This coforall-loop is iterating over the `range` 0..#numTasks which
```

Comment:
0..#numTasks might looks better as inline code `0..#numTasks` (same with the writeln() below)